Lib/test/test_ast.py

```diff
@@ -13,7 +13,7 @@
 from test import support
 
 def to_tuple(t):
-    if t is None or isinstance(t, (str, int, complex)):
+    if t is None or isinstance(t, (str, int, complex)) or t is Ellipsis:
```

Comment:
```suggestion
    if t is None or isinstance(t, (str, int, complex, types.EllipsisType)):
```

Reply:
The current version seems cleaner to me, and easier to port to other Python versions.